xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -581,6 +581,9 @@ void CDVDDemuxFFmpeg::Reset()
 
 void CDVDDemuxFFmpeg::Flush()
 {
+  if (m_ioContext)
```

Comment:
test m_pFormatContext->pb in the block below. we don't create the ioContext for all cases ourselves.